playbooks/roles/learner-profile-dashboard/tasks/deploy-app.yml

```diff
@@ -0,0 +1,51 @@
+---
+
+- name: checkout repository
+  git:
+    repo: "{{ LPD_REPOSITORY }}"
+    version: "{{ LPD_REPOSITORY_VERSION }}"
+    dest: "{{ LPD_APP_LOCATION }}"
+
+- name: install dependencies
+  pip:
+    virtualenv: "{{ LPD_VENV }}"
+    state: present
+    requirements: "{{ LPD_APP_LOCATION }}/requirements/base.txt"
+    virtualenv_python: "/usr/bin/python2.7"
+
+# This is a hack Gunicorn==17.5 is installed system wide, so virtualenv and system version match.
+# possibly this should be sloved in a cleaner way
```

Comment:
@tomaszgy This comment should mention why we need a different version of `Gunicorn`.

Also, a couple typos to fix are:

* Missing `:` after `This is a hack`
* Capitalize `possibly`
* `sloved` should be `solved` 😄

Reply:
@itsjeyd, please see the current version of the comment.